python/pyspark/__init__.py

```diff
@@ -46,10 +46,12 @@
 from pyspark.broadcast import Broadcast
 from pyspark.serializers import MarshalSerializer, PickleSerializer
 
-# for back compatibility
-from pyspark.sql import SQLContext, HiveContext, SchemaRDD, Row
+from pyspark.graphx.vertex import VertexRDD
+from pyspark.graphx.edge import EdgeRDD, Edge
+from pyspark.graphx.graph import Graph
 
 __all__ = [
     "SparkConf", "SparkContext", "SparkFiles", "RDD", "StorageLevel", "Broadcast",
     "Accumulator", "AccumulatorParam", "MarshalSerializer", "PickleSerializer",
-]
+    "VertexRDD", "EdgeRDD", "Edge", "Graph"]
```

Comment:
Similarly, I'm not sure that we want to add the GraphX methods to `__all__` here since it doesn't include the SQL ones.